install.bash

```diff
@@ -54,3 +55,9 @@ git clone https://github.com/mattmezza/dotfiles.git ~/dotfiles
 
 vim +PluginInstall +qall
 
+git config --global alias.co checkout
```

Comment:
these are great, they seem unrelated though?

Reply:
Sorry dude, the PR is kinda old, I pushed to my master that's why you got the notification from GitHub. Anyways, git aliases might be unrelated but it happens often to my to write `git st` and getting the `command not found` cause there is no git alias on the machine. This is the main reason why I added it.

Reply:
yeah, totally. this is cool, i was just trying to figure out if it was related to anything else